### PR TITLE
Prime_R

### PR DESCRIPTION
**Pull request by rooski15**
Begun the process of adding Prime_R keyboard. 

I believe I have all of the necessary items from the keyboard template, as well as the default layout and my own. I did not design / make the keyboard, I've just attempted to develop a QMK package from the .json.

That said, I welcome any feedback or constructive criticism - this is my first contribution.

Also included a couple of my own keymaps unintentionally.

Cheers,
Andrew

### CODE REVIEW

File: keyboards/prime_r/config.h [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6060
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    qmkbuilder
+#define PRODUCT         keyboard
+#define DESCRIPTION     Keyboard
```
Comment (jackhumbert) [inline keyboards/prime_r/config.h]: Would you mind filling these lines out to represent the keyboard?

Comment (rooski15) [reply keyboards/prime_r/config.h]: Done.  Unsure on device_ver, believe it is 1.0

Comment (rooski15) [reply keyboards/prime_r/config.h]: @jackhumbert - I read up on the device ID information - I'm not sure the manufacturer has gone through the registration process, but I will reach out to them and see.

If they have not, what would you have me do for the vendor / product ID?

Comment (jackhumbert) [reply keyboards/prime_r/config.h]: You can leave the vendor/product ID as the bogus ones we use right now - we're working on a deeper solution for this :)

Comment (rooski15) [reply keyboards/prime_r/config.h]: Awesome. :)

**Comment by jackhumbert** [comment]
Cool! Thanks :)

**Pull request merged**